content/plugins/gcp.md

```diff
@@ -278,7 +304,7 @@ mx:
 
 {{< /gsHighlight >}}
 
-The DNSRecord type can be connected to an instance or directly to an IP. In this case the (associated) public IP will be added to the list of resources.
+The DNSRecord type can be connected to an instance, or directly to an IP address. In the case of an IP address, the (associated) public IP is added to the list of resources.
```

Comment:
(revised to add the relationships that must be used) "The DNSRecord The DNSRecord type can be connected to a `cloudify.gcp.nodes.Instance` (using `cloudify.gcp.relationships.dns_record_connected_to_instance`), or to a `cloudify.gcp.nodes.IPAddress` (using `cloudify.gcp.relationships.dns_record_connected_to_ip`). Each instance or IP address node that is connected will have its public IP address added to the list of resources for this record."